docker/chunkflow/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN pip install -r /igneous/requirements.txt
 RUN pip install pyasn1 --upgrade
 RUN cd /igneous && pip install -e .
 
-CMD python /igneous/igneous/task_execution.py
+#CMD python /igneous/igneous/task_execution.py
```

Comment:
Not sure what's going on here? The igneous pipeline requires this to function.

Reply:
I need to add `CMD python /igneous/igneous/task_execution.py --no-loop` option for my docker run. I used error handling in `user_data`.